packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -6,6 +6,149 @@ section: get-started
+
+- Expandable section 
+    - **Example:** All [expandable sections](/components/expandable-section) in our examples and demos.
+
+- Form
+    - **Requires opt-in.**
+    - **Example:** [Form field groups](/components/forms/form#field-groups), which have opted in.  
+
+- Navigation
+    - **Example:** [Expandable navigation examples](/components/navigation#expandable).
+    - **Note:** This animation was introduced in our 6.2 release, but performance issues were reported. For this release, we reworked the animation to resolve these issues.
+
+- Search input   
+    - **Requires opt-in.**
+    - **Example:** [Expandable search input](/components/search-input#with-expandable-button), which has opted in.
+
+- Tree view
+    - **Requires opt-in.**
+    - **Example:** All [tree view](/components/tree-view) examples and demos, which have opted in. 
+
+- Table
+    - **Requires opt-in.**
+    - **Example:** [Expandable table](/components/table#expandable), which has opted in.
+    - **Note:** This release does not include expansion animations for tree tables.
+
+#### Feedback 
+
+The following components contain animations that give users dynamic feedback, helping to communicate that an action has taken or will take place.  
+
+1.  Buttons
+    - **Animation:** When a button is clicked, the active state color ripples outward from the center of the button.
+    - **Example:** All buttons in our components, including in our [button examples and demos](/components/button).
+
+1.  Form validation
+    - **Animation:** When an error occurs in a form field with validation, the component "jiggles" from side to side as danger styles are applied. 
+    - **Example:** The [validated form select](/components/forms/form-select#validated), [invalid text area](/components/forms/text-area#invalid), and [invalid text input](/components/forms/text-input#invalid) examples.
+
+1.  Menu toggle validation
+    - **Animation:** When an error occurs in a menu toggle with validation, the component "jiggles" from side to side as danger styles are applied. 
+    - **Example:** The [danger option in the select with validation](/components/menus/select#with-validation) example.
+
+1.  Progress
+    - **Animation:** When an error occurs in a progress bar, the component "jiggles" from side to side as danger styles are applied. 
+    - **Example:** The "Danger variant" option in our [progress helper text example](/components/progress#helper-text).
+
+1.  Tabs 
+    - **Requires opt-in for HTML-only implementations.**
+    - **Animation:** The bottom "active tab" line slides between tabs as they are selected.
+    - **Example:** All [default tabs](/components/tabs) in our examples and demos. This animation does not apply to boxed tabs.
+    - **Note:** To avoid a breaking change, this animation is opt-in for HTML-only implementations. To support this, view our [HTML guidance for animating the tab accent mark](/components/tabs#animate-current-tab-accent).
+
+##### Icons
```

Comment:
Maybe worth calling out, but for these icon animations, currently there's no ability to customize the icon used. So if the consumer is passing isFavorite, isHamburger, or isSettings, they're stuck with the icon for that respective button.